### PR TITLE
✨ Add checkbox list as a field type

### PR DESCRIPTION
**Pull request by Sven65**
Adds a new checkbox list component according to #CU-234a8f5

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-234a8f5 Skapa ny BulletList komponent med checkbox i formbuildern](https://app.clickup.com/t/234a8f5)

File: src/preview/components/CheckboxList/CheckboxList.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  text: PropTypes.string,
+  /**
+   * sets the color theme.
+   */
+  color: PropTypes.oneOf(['blue', 'green', 'red', 'purple']),
+  /**
+   * One of small, medium, large
+   */
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+};
```
Comment (MasterKenth) [inline src/preview/components/CheckboxList/CheckboxList.tsx:50]: Do we still need `propTypes` if we're using a Typescript interface? (Some components seems to have it but others don't so I don't know.)

**Review by MasterKenth — COMMENTED** [review]
I have some questions, but otherwise it works.

File: src/preview/components/CheckboxList/CheckboxList.tsx (hunk 1 above)
Comment (Sven65) [reply src/preview/components/CheckboxList/CheckboxList.tsx]: I'd vote for removing proptypes where we can. I agree that they're redundant when we're using TS interfaces.

**Review by MasterKenth — APPROVED** [review]

**Pull request merged**